### DISCUSSION TITLE
Installation Documentation

### DISCUSSION THREAD

**Discussion by mogui** [dexidp/dex#2375] [Q&A] [answered]
Sorry to ask that, but maybe it's me, I cannot find a piece of docuemtnation describing how to install dex, I found an helm chart repository under the organization, but not doceumnted too.

Am I missing something, or I have to figure out by my own ?

**Comment by sagikazarmark** [comment, marked as answer]
The instructions in the Helm chart should help you get started: https://github.com/dexidp/helm-charts/tree/master/charts/dex

More detailed instructions about configuring Dex is available on our website: https://dexidp.io/docs/

Let us know if you have any specific issue regarding installation. The docs are not perfect, so any feedback could help us improve it.

**Reply by mogui** [reply]
Thanks, I saw the  documentation and regarding configuration i think it's ok.
I think it's missing a straightforward "How to install" pointing to the helm chart instruction and maybe other way to ionstall.
Just to say.
Anyway thanks

**Reply by bmhkb4** [reply]
I agree with mogui, the instructions on https://dexidp.io/docs/ are not clear on either a plain minimum container deployment and there are no instructions on helm deploy link (https://charts.dexidp.io/) on how to actually use it.  This thread helped me find the actual minimum instructions to deploy dex.

**Reply by bmhkb4** [reply]
also this has no instructions -> https://github.com/dexidp/helm-charts

**Reply by bmhkb4** [reply]
the [example config](https://github.com/dexidp/dex/blob/master/examples/config-dev.yaml) on the main docs page says it is deprecated too.  Probably need to update those config links.

**Comment by esmithaas** [comment]
The documentation is terrible in regards to how to deploy.

**Comment by Kube-spec** [comment]
Is it possible to install Dex in Ubuntu(22.04) VM. If yes kindly provide the simple steps how to install it.
As I have gone through the https://dexidp.io/docs/ and https://github.com/dexidp/dex/blob/master/README.md not able to figure out how to install it. There are different installation and configuration has been mentioned as per different scenario but for me it is difficult to understand from the document. 
Could you please help in this regards.
